src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/TintColor/IconTintColorBehavior.shared.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.Versioning;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+/// <summary>
+/// A behavior that allows you to tint an icon with a specified <see cref="Color"/>.
+/// </summary>
+[UnsupportedOSPlatform("Windows")]
+public partial class IconTintColorBehavior
+#if !(ANDROID || IOS || MACCATALYST)
+ : PlatformBehavior<View>
+#endif
+{
+	/// <summary>
+	/// Attached Bindable Property for the <see cref="TintColor"/>.
+	/// </summary>
+	public static readonly BindableProperty TintColorProperty =
+		BindableProperty.Create(nameof(TintColor), typeof(Color), typeof(IconTintColorBehavior), default);
+
+	/// <summary>
+	/// Property that represents the <see cref="Color"/> that Icon will be tinted.
+	/// </summary>
+	public Color? TintColor
+	{
+		get => (Color?)GetValue(TintColorProperty);
+		set => SetValue(TintColorProperty, value);
+	}
```

Comment:
`null` is a value that we want here... Because if the dev sets the `Color` to `null`, that means he/she wants to clear the tint color from that control.